wikitemplate-minorCRbumpDesktop.md

```diff
@@ -6,7 +6,11 @@
 ### Widevine
 
 - [ ]  Verify `Widevine Notification` is shown when you visit Netflix for the first time
-- [ ]  Test that you can stream on Netflix on a fresh profile after installing Widevine 
+- [ ]  Test that you can stream on Netflix on a fresh profile after installing Widevine
+- [ ]  Verify `Widevine Notification` is shown when you visit HBO Max for the first time
+- [ ]  Test that you can stream on HBO Max on a fresh profile after installing Widevine
+
+
```

Comment:
just a small nitpick: we can probably remove one of the extra spaces as there's 3 between the ending of the Widevine test and the start of the `### Rewards` title.

Reply:
Yep, thanks; fixed.